python/taichi/lang/impl.py

```diff
@@ -158,6 +158,25 @@ def subscript(value, *indices):
             ])
         else:
             return Expr(_ti_core.subscript(var, indices_expr_group))
+    elif isinstance(value, AnyArray):
+        field_dim = int(value.ptr.get_attribute("dim"))
```

Comment:
nit: Leave a TODO so that we don't do this..?

Reply:
Do you mean that we are going to put `subscript()` in their individual classes?

Reply:
sorry, i mean the "dim" part

Reply:
done